nengo_viz/static/viz_netgraph.js

```diff
@@ -104,6 +131,7 @@ VIZ.NetGraph = function(parent, args) {
             self.offsetY = (self.offsetY + yy) / scale - yy;
             
             self.scale = scale * self.scale;
+            interact.margin(30*self.scale);
```

Comment:
I'm more than happy to take that out. It was an experiment.  The downside of removing it is that corner drags disappear for scaled up ensembles, but that might be less annoying.

On 23 May 2015 11:29:15 GMT-04:00, tcstewar <notifications@github.com> wrote:
>> @@ -104,6 +131,7 @@ VIZ.NetGraph = function(parent, args) {
>>              self.offsetY = (self.offsetY + yy) / scale - yy;
>>              
>>              self.scale = scale * self.scale;
>> +            interact.margin(30*self.scale);
>
>That's true if "super small" means "super small on the screen", and I'd
>agree that in that case being able to drag using the text is important.
>However, here, the problem is that if the object is "super small" in
>terms of size in the model, but you've zoomed in such that it takes up
>50% of your screen, then it's *still* not moveable.  
>
>My feeling is that, as far as the user interface goes, the width of the
>"resize" area should be fixed in terms of pixels, and not change
>depending on your global zoom level.  That way if something is small in
>the absolute sense, but you're zoomed in near it, then you can still
>move it as expected.  Otherwise, with *any* mildly complicated model,
>you're never going to be able to move anything by dragging on the item,
>you'll always have to move it via the text (or some other way).
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/nengo/nengo_viz/pull/180/files#r30945714